src/components/SidebarContent.vue

```diff
@@ -15,6 +15,7 @@
           class="button"
           @click="searchEvent"
           size="large"
+          :disabled="!searchInput.trim()"
```

Comment:
@alan-wu, yes, the list should be sorted from newest to oldest, and I've updated it. 
For the search disabled, I tried to prevent searching empty search terms, but since the users might want to clear/reset the search, the button shouldn't be disabled. I set it back to enable state.